packages/libcap.rb

```diff
@@ -24,22 +16,25 @@ class Libcap < Package
   def self.patch
     # add includes option to make it work with gperf-3.1
     system 'sed -e "/gperf --/s/gperf --/gperf --includes --/" -e "/gperf --/s/cap_lookup_name(/cap_dummy(/" -i libcap/Makefile'
-    # change the path to ld
-    system "sed -i 's,/usr/bin/ld,#{CREW_PREFIX}/bin/ld,g' Make.Rules"
+
+    # change the path to mold
+    system "sed -i 's,/usr/bin/ld,#{CREW_PREFIX}/bin/mold,g' Make.Rules"
+
     # change prefix
     system "sed -i 's,prefix=/usr,prefix=#{CREW_PREFIX},g' Make.Rules"
+
     # set exec_prefix
     system "sed -i 's,^exec_prefix=,exec_prefix=\$(prefix),g' Make.Rules"
-    if ARCH == 'x86_64'
-      # use system libdir
-      system "sed -i 's,^lib_prefix=\$(exec_prefix),lib_prefix=#{CREW_LIB_PREFIX},g' Make.Rules"
-    end
+
+    # use system libdir
+    system "sed -i 's,^lib_prefix=\$(exec_prefix),lib_prefix=#{CREW_LIB_PREFIX},g' Make.Rules"
+
     # http://git.yoctoproject.org/cgit.cgi/poky/plain/meta/recipes-support/libcap/files/0001-ensure-the-XATTR_NAME_CAPS-is-defined-when-it-is-use.patch
     system 'sed -i "s,^\#ifdef VFS_CAP_U32,\#if defined (VFS_CAP_U32) \&\& defined (XATTR_NAME_CAPS),g" libcap/cap_file.c'
```

Comment:
A fix for this issue seems to have been upstreamed a few years ago: https://bugzilla.kernel.org/show_bug.cgi?id=208375